src/snowcli/utils/threadsafe.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, value: T):
         self._value = value
         self._lock = threading.Lock()
 
-    def set(self, new_value: T) -> T:
+    def set(self, new_value: T) -> T:  # noqa: A003
```

Comment:
I'm wondering what is the source of this `A003` warning?
Neither flake8, pylint nor mypy reports this issue.

```
snowcli on  SNOW-986249-option-for-intermediate-output-of-command [$!] via 🐍 v3.8.18 (py3.8) took 13s
❯ flake8 src/snowcli/utils/threadsafe.py

snowcli on  SNOW-986249-option-for-intermediate-output-of-command [$!] via 🐍 v3.8.18 (py3.8)
❯ mypy !$
mypy src/snowcli/utils/threadsafe.py
Success: no issues found in 1 source file

snowcli on  SNOW-986249-option-for-intermediate-output-of-command [$!] via 🐍 v3.8.18 (py3.8)
❯ pylint !$
pylint src/snowcli/utils/threadsafe.py
************* Module src.snowcli.utils.threadsafe
```

_pylint only complains about docstrings_

Reply:
https://pypi.org/project/flake8-builtins/
```
A003 Class attribute `set` is shadowing a Python builtin
```